python-project-template/pyproject.toml.jinja

```diff
@@ -58,6 +58,10 @@ dev = [
 {%- endif %}
 ]
 
+[metadata]
+long_description = { file = "README.md" }
```

Comment:
What is `long_description` used by? Is it picked up by PyPI? I've seen it around in other pyproject.toml files, but it's not clear what it adds. I also don't see it listed in the pyproject.toml schema here: https://validate-pyproject.readthedocs.io/en/latest/schemas.html (but this schema could be incomplete)